### PR TITLE
Jetpack Cloud: Scanner display threats

### PR DESCRIPTION
**Pull request by enejb**
#### Changes proposed in this Pull Request

This PR attempts to render the treats as best as it can. 
We should move most of the treats details to the api vs figuring here. 
I reused quite a bit of code from treats-alert.jsx component to figure out nicer nameing etc. 

After:
Screen Shot 2020-04-17 at 12 04 37 PM

Bonus part here is that treat ignoring works as expected 🥳 for the treat that I ignored. 

#### Testing instructions
* On a site that has treats visit 
http://jetpack.cloud.localhost:3000/scan/{example.com}

Notice the sections renders real threats.

### CODE REVIEW

File: client/landing/jetpack-cloud/components/threat-item/index.tsx [hunk 1]
```diff
@@ -48,29 +50,169 @@ class ThreatItem extends Component< Props > {
...
+			case 'theme':
+				return translate( 'Vulnerable Theme {{themeSlug/}} (version {{version/}})', {
+					components: {
+						themeSlug: <span className="threat-item__alert-slug">{ extension.slug }</span>,
+						version: <span className="threat-item__alert-version">{ extension.version }</span>,
+					},
+				} );
+
+			case 'database':
+				return translate( 'Database %(threatCount)d threat', 'Database %(threatCount)d threats', {
```
Comment (a8ci18n) [inline client/landing/jetpack-cloud/components/threat-item/index.tsx:144]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Jetpack identified %(threatCount)d threat in your database.', 'Jetpack identified %(threatCount)d threats in your database.' )` *ES Score: 7*

File: client/landing/jetpack-cloud/components/threat-item/index.tsx [hunk 2]
```diff
@@ -12,6 +12,8 @@ import { Button } from '@automattic/components';
 import LogItem from '../log-item';
 import ThreatDescription from '../threat-description';
 import { Threat } from 'landing/jetpack-cloud/components/threat-item/types';
```
Comment (rcanepa) [inline client/landing/jetpack-cloud/components/threat-item/index.tsx:14]: We can tackle this in another PR when we have more clarity of the final shape of the threat object, but we should include those changes in the Threat type.

**Comment by ChaosExAnima** [comment]
Taking this over. Going to get typing updated and test with a bunch of threats.

**Comment by enejb** [comment]
This is working really great! Thanks for all the fixed @ChaosExAnima !

**Comment by rcanepa** [comment]
I just pushed a small update that changes the positioning of the buttons on mobile.
image

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3324647

Thank you @enejb for including a screenshot in the description! This is really helpful for our translators.

**Pull request merged**